src/test/java/Homework17.java

```diff
@@ -0,0 +1,60 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest {
+
+    @Test
+    public void addSongToPlaylist() {
+      openLoginUrl();
+      login();
+      createPlaylist();
+      clickAllSongs();
+      addFirstSongToPlaylist();
+      Assert.assertTrue(msgPopup());
+    }
+
+public boolean msgPopup(){
+        WebElement msg = driver.findElement(By.cssSelector("div[class='success show']"));
+        return(msg.isDisplayed());
+
+}
+public void createPlaylist() {
+    WebElement playlistbtn = driver.findElement(By.cssSelector("i[title='Create a new playlist']"));
+    playlistbtn.click();
+    WebElement playlistcontextmenu = driver.findElement(By.cssSelector("li[data-testid='playlist-context-menu-create-simple']"));
+    playlistcontextmenu.click();
+    WebElement inputplaylistname = driver.findElement(By.cssSelector("input[name='name']"));
+    inputplaylistname.sendKeys("Test");
+    inputplaylistname.sendKeys(Keys.ENTER);
+
+}
+public void searchinput(){
+        //input[type='search']
+    WebElement searchfield = driver.findElement(By.cssSelector("input[type='search']"));
+    searchfield.click();
+    searchfield.sendKeys("dark");
+    searchfield.sendKeys(Keys.ENTER);
+    WebElement allsongsbnt = driver.findElement(By.cssSelector("button[data-test='view-all-songs-btn']"));
+    allsongsbnt.click();
+
+}
+    public void clickAllSongs() {
+        WebElement allsongs = driver.findElement(By.cssSelector("a[href='#!/songs']"));
+        allsongs.click();
+    }
+    public void addFirstSongToPlaylist(){
+        WebElement song = driver.findElement(By.xpath("//tr/td[4]"));
```

Comment:
You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/222373781-50f4a762-8a8e-4197-b8e8-4f3550507e7c.png)

Hint

if you are beginner does not know who to write the xpath then you can relay on the xpath that will be provided by chrome browser. Right click on element that you are interested in then click on Inspect. Now right click on the html code of the element Copy > Copy Xpath. Now you have the xpath in the clipboard for you.

_For example, here is the locator (Copy > Copy Xpath):_  `//*[@id="sidebar"]/section[1]/ul/li[3]/a`

**BUT I recommend not using plugins or copying the xpath from the page code using a browser or web developer.**
For example, here is the correct locator: ` //a[@class='songs']`

It is important to mention that the following principle must be held in deciding locators: the locator must be

- unique
- readable 
- maintainable